shell/platform/darwin/macos/framework/Source/FlutterEngine.mm

```diff
@@ -280,7 +279,8 @@ - (BOOL)runWithEntrypoint:(NSString*)entrypoint {
   }
 
   [self sendUserLocales];
-  [self updateWindowMetrics];
+
+  [self.viewController.flutterView start];
```

Comment:
Is there a reason to have `start` rather than doing it in `init` or `ctor` for `FlutterView`?

Reply:
IIRC start causes flutterView to drive viewport size change. I think I'd prefer that to be separate method rather than doing it from constructor.